frontend/packages/console-shared/src/components/lists/DetailPropertyListItem.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { ListItem } from '@patternfly/react-core';
+
+type Props = {
```

Comment:
can you name this better - such as `DetailPropertyListItemProps` and move to bottom as we usually do for type declarations ?

Reply:
I've renamed the props but I'd prefer to keep them on top. Both conventions are used in console and I find putting props to the top more convenient as it's not necessary to scroll over the component body to see the props.